addOns/pscanrules/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
+### Added
+- Added Trusted Domains in rule 10017 (Issue 7775)
```

Comment:
- `rule 10017` → `Cross-Domain JavaScript Source File Inclusion`.
- Add dot at the end.

Reply:
I meant replace the former withe the latter, i.e. `Added Trusted Domains in Cross-Domain JavaScript Source File Inclusion (Issue 7775).`